### DISCUSSION TITLE
Revert a Cronos transaction

### DISCUSSION THREAD

**Discussion by ipaidimarla** [crypto-org-chain/cronos#746] [Q&A]
Hi,
I sent my USDC to wrong network type (Cronos instead of ERC-20).
My funds are missing now. How do I revert this transaction?
I contacted support@cronos.org and product@cronos.org. Both these emails bounced back to me.

**Comment by ipaidimarla** [comment]
0xf604dd030bea3ef518bb836564100b43c4b3d8bfd8ec391d2edf8853a53e6d33. here is the hash

**Comment by aw126** [comment]
Hi, could you please provide more details about the transaction? Such as where you sent the funds to, which wallet you used to perform the transaction, etc. 
For email inquiries, you may send your questions to chain@crypto.org next time.

**Comment by aw126** [comment]
Based on the receiving address (https://cronoscan.com/address/0xc600d6b4e5e7d45e7c3e579353568447d877bcda), it looks like there are only CRC20 tokens without any CRO as fee to further retrieve the USDC. The possible solution is to get some CROs and transfer them to the address via the Cronos network based on the current info.
